util/c_function.py

```
@@ -60,23 +60,26 @@ def __init__(
+        self.source_code_with_specs = ""
+        self._cached_source_code = ""
         self.arg_names = []
         self.arg_types = []
         self.enums = []
         self.global_vars = []
         self.structs = []
 
-    def get_original_source_code(
+    def get_source_code(
         self,
         include_documentation_comments: bool = False,
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)